dev/dll/XamlControlsResources.cpp

```diff
@@ -197,3 +200,33 @@ void SetDefaultStyleKeyWorker(winrt::IControlProtected const& controlProtected,
     }
 #endif
 }
+
+// Normally global reveal lights are attached automtically via the first call to
```

Comment:
This all had to move as a result of deleting ThemeResources?

Reply:
yes. I have to rename ThemeResources to XamlControlsResources, and also remove ThemeResourcesFactory.
EnsureRevealLights are from ThemeResourcesFactory.cpp